seatunnel-e2e/seatunnel-spark-connector-v2-e2e/connector-file-spark-e2e/src/test/resources/file/fakesource_to_local_text.conf

```diff
@@ -54,12 +82,8 @@ sink {
     path="/tmp/test/text"
     field_delimiter="\t"
     row_delimiter="\n"
-    partition_by=["age"]
-    partition_dir_expression="${k0}=${v0}"
-    is_partition_field_write_in_file=true
```

Comment:
Use update instead of remove?

Reply:
These parameters are not applicable in this scenario. So I removed those. The core of the test is full type writing. What do you think about it?

Reply:
ok, maybe there can be dedicated partition testcase

Reply:
Yup, I will add more test cases in next pull requests until the connector is stable.